Solutions/Recorded Future/Playbooks/RecordedFuture-IP-Actively_Communicating_C2_Server-IndicatorProcessor.json

```diff
@@ -0,0 +1,192 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"metadata": {
+		"comment": "This playbook leverages the Recorded Future API and automatically imports the Actively Communicating C&C Server IP RiskList, as Threat Intelligence Indicators, for detection purposes in Azure Sentinel",
+		"author": "Adrian Porcescu, Recorded Future"
+	},
+	"parameters": {
+		"PlaybookName": {
+			"defaultValue": "RecordedFuture-IP-Actively_Communicating_C2_Server-IndicatorProcessor",
+			"type": "String"
+		},
+		"PlaybookNameBatching": {
+			"defaultValue": "RecordedFuture-ImportToSentinel",
+			"type": "String"
+		}
+	},
+	"variables": {
+		"RecordedFutureConnectionName": "[concat('recordedfuture-', parameters('PlaybookName'))]"
```

Comment:
Please reduce the playbook name. As after concatenation it's become more than 80 chars which is not acceptable during deployment.